nirnroute.html

```diff
@@ -94,6 +80,26 @@ <h2>Close image</h2>
 </div>
 </div>
 
+<div class="resizer" id="dragMe"></div>
+
+<div class="sidePanel">
+<div class="contentContainer">
+	<div class="sidebarNirnroute" id="sidebar">
+	<!-- sidebar contains images n navigation n stuff.-->
+	<h1 id="nirnName">Nirnroot Title</h1> <span class= "h3" >Nirnroots Found: <span id="overview_nirnroot">##</span></span>
+	<br/>
+	<button id="prevButton" title="left arrow">Prev</button><button id="nextButton" title="right arrow">Next</button>&nbsp;&nbsp;<label>Jump to <input type="number" id="nirnIdField"/></label>
```

Comment:
I'll remove the duplicate space for now. The css for that nirnroute info box is long overdue for an overhaul in general (I don't think we've revisited its layout since the feature was first implemented) but I was going to wait until this PR is done before moving on to that.